system/core/pages.php

```diff
@@ -187,6 +187,11 @@ function loadPage(RequestContext $O_O) {
 						break;
 				}
 				break;
+			case HyphaRequest::HYPHA_SYSTEM_PAGE_HELP:
+				$subject = isset($args[0]) ? urldecode($args[0]) : 'undefined';
```

Comment:
I wonder if this is the right place to urldecode? Shouldn't HyphaRequest perhaps handle that?

Reply:
I created #203 for a proper solution (which requires more thought and testing), for now let's leave it here.